medicaltorch/datasets.py

```diff
@@ -63,27 +65,32 @@ class SegmentationPair2D(object):
 
     :param input_filename: the input filename (supported by nibabel).
     :param gt_filename: the ground-truth filename.
+    :param metadata: metadata related to image
+    :param contrast: contrast of image (e.g. T2w)
```

Comment:
Are you using contrast information to further organise your channels?
If so, I would recommend to call it `channel` --> more generic, so that it could be integrated later into `perone/medicaltorch`.